test/JDBC/expected/dbcreator_role-vu-verify.out

```diff
@@ -1,4 +1,16 @@
 -- tsql
+-- dbcreator should not have membership in database guest role
+SELECT r.rolname AS parent_role, mr.rolname as member 
+FROM pg_auth_members m 
+JOIN pg_roles r ON (m.roleid = r.oid) 
+JOIN pg_roles mr ON (m.member = mr.oid) 
+where mr.rolname = 'dbcreator';
```

Comment:
suggestion: alternative smaller query we can execute in psql endpoint `SELECT roleid::regrole, member::regrole FROM pg_auth_members WHERE member = 'dbcreator'::regrole;`

Reply:
lets also runs this query for sysadmin

Reply:
We haven't handled any change for sysadmin in this which means it will continue to the membership as earlier.